GPU/Common/PresentationCommon.h

```diff
@@ -31,7 +31,12 @@ struct CardboardSettings {
 struct PostShaderUniforms {
 	float texelDelta[2]; float pixelDelta[2];
 	float time[4];
-	float video; float pad[3];
+	float video;
+	float setting1;
```

Comment:
This uses up a lot of slots on Direct3D 9.  I'd suggest instead using `u_setting` and have the four settings be `u_setting.x`, `u_setting.y`, etc.  This might also perform a tiny bit better on GLES, I especially suspect Mali-4xx with its register limits.

To make alignment simpler, I would put the new `[4]` before video.  The reason `pad` was on the same line as video before is that the GPU strictly requires this data to be aligned to 16 bytes, in sets of 4 floats.  So even if we keep it as setting1, setting2, we should line them up.

-[Unknown]